src/api/app/models/event/base.rb

```diff
@@ -171,7 +171,13 @@ def perform_create_jobs
     end
 
     def send_event_emails_job
-      SendEventEmailsJob.perform_later(id) if id.present?
+      return if id.blank?
+
+      if subscribers.empty?
```

Comment:
this might pass test suites, but this moves the quite heavy subscribers call into the event generation - basically turning the 'fetch jobs from backend' into slow motion - as such you move the most complex part of the job from the part that can be parallelized to a serial part.

Reply:
You are right. I am closing the PR then.

Thanks for the heads up